src/main/java/nl/igorski/example/MWEngineActivity.java

```diff
@@ -121,7 +121,10 @@ private void init()
         // add a filter to synth 1
         maxFilterCutoff = ( float ) SAMPLE_RATE / 8;
 
-        _filter = new Filter( maxFilterCutoff / 2, ( float ) ( Math.sqrt( 1 ) / 2 ), minFilterCutoff, maxFilterCutoff, 0f, 1 );
+        _filter = new Filter(
+            maxFilterCutoff / 2, ( float ) ( Math.sqrt( 1 ) / 2 ),
+            minFilterCutoff, maxFilterCutoff, 0f, OUTPUT_CHANNELS
+        );
```

Comment:
Changed hard coded _1_ to reflect the amount of channels the engine is configured to output.